pull/github_context.go

```diff
@@ -224,6 +224,25 @@ func (ghc *GithubContext) CurrentSuccessStatuses(ctx context.Context) ([]string,
 			opts.Page = res.NextPage
 		}
 
+		checkOpts := &github.ListCheckRunsOptions{ListOptions: github.ListOptions{PerPage: 100}}
+		for {
+			checkRuns, res, err := ghc.client.Checks.ListCheckRunsForRef(ctx, ghc.owner, ghc.repo, ghc.pr.GetHead().GetSHA(), checkOpts)
```

Comment:
Does this returns an empty list when there are no check runs for the ref or a 404? I know there are some GitHub APIs that behave one way and some that behave the other.

Reply:
Looks like it returns an empty array.

```console
> curl -H 'Accept: application/vnd.github.antiope-preview+json' 'https://api.github.com/repos/palantir/bulldozer/commits/08efe6866fe81693f6da541cc4575a49444b656a/check-runs?per_page=100'
{
  "total_count": 0,
  "check_runs": [

  ]
}
```